src/Bravo3/OrmBundle/DependencyInjection/OrmExtension.php

```diff
@@ -37,6 +37,8 @@ public function load(array $configs, ContainerBuilder $container)
             $em->setFactory([self::ORM_FACT_CLASS, self::ORM_FACT_METHOD]);
         }
 
+        $container->setParameter('hydration_exceptions_as_events', $config['hydration_exceptions_as_events']);
```

Comment:
This should be:
`$container->getDefinition('orm.em')->addArgument($config['hydration_exceptions_as_events']);`

And leave off the parameter in the services.yml file